package.json

```diff
@@ -7,11 +7,20 @@
     "node": ">=10.15.0"
   },
   "main": "index.js",
-  "playwright": {
-    "chromium_revision": "762211",
-    "firefox_revision": "1087",
-    "webkit_revision": "1211"
-  },
+  "browsers": [
```

Comment:
how do you feel about moving browser config out to a separate file? This way we'd avoid abusing
`package.json`

Reply:
My understanding is that this is what package.json is for :)